src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -261,22 +261,14 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             // if an arg parser was registered, but failed, bail
             if (pfn && args == nullptr)
             {
-                return nullptr;
+                return make_self<ActionAndArgs>();
             }
         }
 
-        if (action != ShortcutAction::Invalid)
-        {
-            auto actionAndArgs = winrt::make_self<ActionAndArgs>();
-            actionAndArgs->Action(action);
-            actionAndArgs->Args(args);
-
-            return actionAndArgs;
-        }
-        else
-        {
-            return nullptr;
-        }
+        // Something like
+        //      { name: "foo", action: "unbound" }
+        // will _remove_ the "foo" command, by returning an "invalid" action here.
+        return make_self<ActionAndArgs>(action, args);
```

Comment:
I don't know how much I love this change...

Reply:
So, here's my reasoning, `ActionAndArgs::FromJson` _could_ use `nullptr` and `Invalid` interchangeably, yes. But then we _always_ have to check for nullness, and we'd be a bit redundant.

This cleans up the code a bit for any consumer _and_ for `ActionAndArgs::FromJson`. 😎